server/src/test/java/org/elasticsearch/plugins/UberModuleClassLoaderTests.java

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.plugins;
+
+import joptsimple.internal.Strings;
```

Comment:
Might be more straightforward to use `org.elasticsearch.common.Strings`?

Reply:
This was a mistake. Thanks for catching it!